config/tgl-cavs.toml

```diff
@@ -227,3 +227,42 @@ count = 8
 	# mod_cfg [PAR_0 PAR_1 PAR_2 PAR_3 IS_BYTES CPS IBS OBS MOD_FLAGS CPC OBLS]
 	mod_cfg = [0, 0, 0, 0, 4096, 100000, 48, 48, 0, 1000, 0]
 
+	[[module.entry]]
+		name = "SRCINTC"
+		uuid = "e61bb28d-149a-4c1f-b709-46823ef5f5ae"
+		affinity_mask = "0xF"
+		instance_count = "10"
+		domain_types = "1"
+		load_type = "0"
+		module_type = "0x7"
+		auto_start = "0"
+		sched_caps = [1, 0x00008000]
+
+	# pin = [dir, type, sample rate, size, container, channel-cfg]
+	pin = [0, 0, 0xffff, 0xc, 0x8, 0x45ff,
```

Comment:
Why are these numbers hex? Hex is OK for bit masks and I/O addresses but not nice for numbers like sample rate.

Reply:
This is used for bit masks. Each bit mask represents whether a certain rate is used. For example, suppose the first bit is to judge whether 16khz is used. Setting 1 means using it and setting 0 means not using it.